### ISSUE TITLE
TypeError: load() missing 1 required positional argument: 'Loader'

### ISSUE THREAD

**Issue by choprahetarth** [VITA-Group/DeblurGANv2#133]
While running the predict.py script on any jpg file the following error pops up - 

```Traceback (most recent call last):
  File "./DeblurGANv2/predict.py", line 126, in <module>
    Fire(main)
  File "./opt/anaconda3/envs/price/lib/python3.9/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "./opt/anaconda3/envs/price/lib/python3.9/site-packages/fire/core.py", line 466, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "./opt/anaconda3/envs/price/lib/python3.9/site-packages/fire/core.py", line 681, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "./DeblurGANv2/predict.py", line 106, in main
    predictor = Predictor(weights_path=weights_path)
  File "./DeblurGANv2/predict.py", line 19, in __init__
    config = yaml.load(cfg)
TypeError: load() missing 1 required positional argument: 'Loader'```

**Comment by lucky-zwx** [comment]
`config = yaml.load(f, Loader=yaml.SafeLoader)`

**Comment by lucky-zwx** [comment]
`config = yaml.load(cfg, Loader=yaml.FullLoader)`

**Comment by choprahetarth** [comment]
> `config = yaml.load(cfg, Loader=yaml.FullLoader)`

Yes, I have corrected this in the #134 pull request
